build/commands/lib/util.js

```diff
@@ -564,7 +564,7 @@ const util = {
     const sig_generator = config.signature_generator
     let src_dir = path.join(config.outputDir, 'signed_binaries')
 
-    if (config.skip_signing || process.env.CERT === undefined || process.env.SIGNTOOL_ARGS === undefined)
+    if (!config.shouldSign())
```

Comment:
This should be equivalent to what was there previously, except for component builds, which I think we do not much care about in this context.

Reply:
not sure what you mean about component builds since those aren't signable. What changes here for component builds?

Reply:
Just the boolean condition. What I have now - `!config.shouldSign()` - always returns `true` for component builds whereas what was there - `config.skip_signing || process.env.CERT ...` - may return `false`. But also in light of what you wrote, I believe `!config.shouldSign()` is what we want anyways.